spec/ParseLiveQuery.spec.js

```diff
@@ -645,6 +645,44 @@ describe('ParseLiveQuery', function () {
     await object.save();
   });
 
+  it('can handle select beforeUnsubscribe trigger', async done => {
+    await reconfigureServer({
+      liveQuery: {
+        classNames: ['TestObject'],
+      },
+      startLiveQueryServer: true,
+      verbose: false,
+      silent: true,
+    });
+    const user = new Parse.User();
+    user.setUsername('username');
+    user.setPassword('password');
+    await user.signUp();
+
+    Parse.Cloud.beforeSubscribe(TestObject, req => {
+      expect(req.requestId).toBe(1);
+      expect(req.user).toBeDefined();
+    });
+
+    Parse.Cloud.beforeUnsubscribe(TestObject, req => {
+      expect(req.requestId).toBe(1);
+      expect(req.user).toBeDefined();
```

Comment:
Can you add more detail here to ensure req.user is correct, such as checking username / req.user.id